README.md

```diff
@@ -147,8 +165,7 @@ cd multicamera_acquisition
 python setup.py develop
```

Comment:
I think we agreed to switch to doing a pip install + making sure all the required packages are specified in setup.py — does that dramatically change anything?

Reply:
Personally I prefer `pip install -e .`

This is what Tim used but I have no problem switching.